README.md

```diff
@@ -82,6 +89,31 @@ Example outputs:
 
 `./ms_data/barcode_results.tsv` - The read pair / amplicon counts for each ORI within each sample.
 
+## Running on input libraries
+
+The pipeline can now be run on input plasmid libraries, not just strains. To do so:
+
+1. first, in the mapping_file, specify "Input" for the sample in the "Strain" column. 
+2. Specify the new library name in the "Library" column. 
+3. Pass a new library_info.csv file containing the metadata for this new library. Crucially, the "Negative control cutoff" field in this library file can/should be empty (it will be ignored) for input files, because these libraries do not have their cutoffs determined yet.
```

Comment:
Can you please provide an explicit example of the `library_info.csv` or add a link to an example file?